readme.md

```diff
@@ -10,44 +10,71 @@ A plugin that makes Prettier organize your imports (i. e. sorts, combines and re
 
 - 👌 Dependency-free (just peer-dependencies you probably already have).
 - 💪 Supports `.js`, `.jsx`, `.ts`, `.tsx` and `.vue` files.
-- 🚀 Zero config.
 - 🤓 No more weird diffs or annoying merge conflicts in PRs caused by import statements.
 - 🤯 If your editor supports auto-imports, you'll stop thinking about your imports so much that you won't even care about their order anymore.
 
 **Caveat**
 
-This plugin inherits, extends and overrides the built-in Prettier parsers for `babel`, `babel-ts`, `typescript` and `vue`, i. e., it's incompatible with other plugins that do the same... so only the last loaded plugin that exports one of those parsers will function.
+This plugin inherits, extends, and overrides the built-in Prettier parsers for `babel`, `babel-ts`, `typescript` and `vue`. This means that it is incompatible with other plugins that do the same; only the last loaded plugin that exports one of those parsers will function.
 
 ## Installation
 
+### Install the Dependency From npm
+
 ```sh
+# If you use npm:
 npm install --save-dev prettier-plugin-organize-imports
+
+# If you use yarn:
+yarn add --dev prettier-plugin-organize-imports
+
+# If you use pnpm:
+pnpm add --save-dev prettier-plugin-organize-imports
 ```
 
-_`prettier` and `typescript` are peer dependencies, so make sure you have those installed in your project._
+Running one of these commands will add `prettier-plugin-organize-imports` to the `devDependencies` section of your project's `package.json` file and download the corresponding files.
 
-## Usage
+Note that `prettier` and `typescript` are peer dependencies, so make sure you also have those installed in your project.
 
-The plugin will be loaded by Prettier automatically. No configuration needed.
+### Add the Plugin to Your Prettier Configuration
 
-Files containing the substring `// organize-imports-ignore` or `// tslint:disable:ordered-imports` are skipped.
+If you do not already have [a Prettier configuration file](https://prettier.io/docs/en/configuration.html), create a file called `prettier.config.mjs` at the root of your project. (While Prettier will work with many different configuration file names, this is the file name that you should choose for modern projects.)
```

Comment:
Ok now this is getting a bit too much, I don't want to repeat the Prettier docs about how to add a config or how to configure plugins here.

I also don't want to tell people how to configure Prettier, which is info that might just be outdated in a couple years.